examples/mnit_ballz/src/ballz_android.nit

```diff
@@ -14,21 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+# Android part of mnit_ballz
 module ballz_android is
-	app_version(1, 0, git_revision)
+	app_version(0, 2, git_revision)
 	app_name("Ballz")
+	app_namespace "org.nitlanguage.ballz"
+	android_api_target 20
+	android_manifest("""<uses-permission android:name="android.permission.INTERACT_ACROSS_USERS" />""")
 end
 
 import android::portrait
-
-import game_logic
+import android::sensors
+import android::audio
+import android::assets_and_resources
```

Comment:
Thanks to #1580, android::audio and android::assets_and_resources should be automatically loaded